src/runtime/type.ts

```diff
@@ -545,9 +545,9 @@ export class BigCollectionType extends Type {
 
 
 export class RelationType extends Type {
-  readonly relationEntities: [Type];
+  readonly relationEntities: [Type, Type, Type];
```

Comment:
This should actually be `Type[]`; RelationTypes can have an arbitrary number of entries.

Reply:
Thanks!  destructuring threw me off.

fixed.